0.2.4 fails to build with GHC 8.6.5 on Windows

Hello! The latest release fails to build with the following error:

```
 System\EasyFile\Missing.hs:180:10: error:
Error:     Variable not in scope:
      setFilePointerEx :: HANDLE -> Integer -> FilePtrDirection -> IO a0
    |
180 |     _ <- setFilePointerEx hdl (fromIntegral siz) fILE_CURRENT
    |          ^^^^^^^^^^^^^^^^
Error: Process completed with exit code 1.
```

This is because `setFilePointerEx` was added in Win32 2.6.2.0 (https://hackage.haskell.org/package/Win32-2.13.2.1/changelog), and GHC versions up to and including 8.10.5 bundle Win32 2.6.1.0. So the latest release can't be built with anything older than 8.10.6, which is quite unfortunate.

Is there any chance this could be fixed?

--- a/System/EasyFile/Missing.hs
+++ b/System/EasyFile/Missing.hs
@@ -172,13 +172,18 @@ getFileSize file = withFileForInfo file $ \fh ->
 getFileSize file = fromIntegral . fileSize <$> getFileStatus file
 #endif
 
+#if (defined(mingw32_HOST_OS) || defined(__MINGW32__))
+-- Earlier versions don't provide `setFilePointerEx`: https://github.com/kazu-yamamoto/easy-file/issues/9
+#if MIN_VERSION_Win32(2, 6, 2)
 -- | Setting the size of the file.
 setFileSize :: FilePath -> Word64 -> IO ()
-#if defined(mingw32_HOST_OS) || defined(__MINGW32__)
 setFileSize file siz = do
     hdl <- createFile file gENERIC_WRITE fILE_SHARE_NONE Nothing oPEN_EXISTING fILE_ATTRIBUTE_NORMAL Nothing
     _ <- setFilePointerEx hdl (fromIntegral siz) fILE_CURRENT
     setEndOfFile hdl
+#endif
 #else
+-- | Setting the size of the file.
+setFileSize :: FilePath -> Word64 -> IO ()
 setFileSize file siz = P.setFileSize file $ fromIntegral siz
 #endif

